### ISSUE TITLE
How to use with Node.JS?

### ISSUE THREAD

**Issue by g992** [partykit/partykit#320]
I see, that in ws.ts file commented WebSocket, and i don't provide it, and get error "WebSocket is not defined"
```
export type Options = {
  // WebSocket?: any;
  maxReconnectionDelay?: number;
  minReconnectionDelay?: number;
  reconnectionDelayGrowFactor?: number;
  minUptime?: number;
  connectionTimeout?: number;
  maxRetries?: number;
  maxEnqueuedMessages?: number;
  startClosed?: boolean;
  debug?: boolean;
};
```

How can i fix it?

**Comment by threepointone** [comment]
Thanks for filing this issue! I have a PR to fix this (#321), we'll land it this week.

**Comment by threepointone** [comment]
Fixed, should go out in the next release tomorrow
